config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/TemplatizedBundle.java

```diff
@@ -28,6 +31,7 @@
 
         private final File newFile;
         private final File originalFile;
+        private final Charset defaultCharSet = StandardCharsets.UTF_8;
```

Comment:
I think we dont need this field, just go for UTF_8 down at 44 and 53 and that's it. 
It's a sonar code smell even minor we can avoid it =)